### ISSUE TITLE
Replace Uptrends for uptime monitoring and status page

### ISSUE THREAD

**Issue by adborden** [GSA/data.gov#1236]
## User Story

As a [user description], I want [behavior description] so that [goal].

## Details
Placeholder story: Uptrends is likely expiring in February 2020. The Tech Portfolio may be working on a solution. 

Uptrends does not provide a public statu6page, but that would be a "nice to have".

## Acceptance Criteria
- [ ]

**Comment by adborden** [comment]
New service is waiting on procurement, should be available after April.

**Comment by hkdctol** [comment]
Closing this as we obtained New Relic Synthetics and asked to unsubscribe to Uptrends alerts through GSA IT.
